kokoro/config/test/ops_agent/release.gcl

```diff
@@ -0,0 +1,25 @@
+import 'common.gcl' as common
+import '../image_lists.gcl' as image_lists
+
+_parse_distro = lambda tag: {
+  parts = split(tag, '/')
+  ret = {
+    distro = parts[3]
+    arch = parts[4]
+  }
+}.ret
+
+config build = common.ops_agent_test {
+  params {
+    local _distro_arch = _parse_distro(params.environment._LOUHI_TAG_NAME)
```

Comment:
This seems very Louhi-specific. Can we express `platforms` in terms of two parameters, `distro` and `arch` (we already have the latter), so it's easy to override them? We can default the values to those extracted from this variable? E.g., something like:
```
config build = common.ops_agent_test {
  params {
    local _distro_arch = _parse_distro(params.environment._LOUHI_TAG_NAME)
    distro = _distro_arch.distro
    arch = _distro_arch.arch
    arch = _parse_distro(params.environment._LOUHI_TAG_NAME).arch
    platforms = image_lists.get("%distro%_%arch%", 'invalid_distro').release
  }
  …
}
```

Reply:
this is covered in the description as being Louhi specific. The intent is to follow up with something similar to what you're asking after it's verified.